tests/utils/esxcli/cmd.go

```diff
@@ -18,15 +18,25 @@ package esxcli
 
 import (
 	"log"
+	"strings"
 
 	"github.com/vmware/docker-volume-vsphere/tests/constants/esxcli"
+	"github.com/vmware/docker-volume-vsphere/tests/utils/misc"
 	"github.com/vmware/docker-volume-vsphere/tests/utils/ssh"
 )
 
+type getVMPowerStatus func(string) string
+
+const (
```

Comment:
As we have discussed before, let's avoid defining global constants in local files. If these constants will be shared, we should define them in the global constants package. If it's just for local use in the same file, we can define them as local variables. Global variables will pollute the package namespace even if they are unexported.